src/types/service_config_test.go

```diff
@@ -8,6 +8,53 @@ import (
 
 var _ = Describe("ServiceConfig", func() {
 
+	Describe("validates required production fields", func() {
+		missingConfig := types.ServiceConfig{}
+
+		publicConfig := types.ServiceConfig{
+			Production: map[string]string{
+				"url": "originate.com",
+			},
+		}
+		privateConfig := types.ServiceConfig{
+			Production: map[string]string{
+				"memory": "128",
+			},
+		}
+		workerConfig := types.ServiceConfig{
+			Production: map[string]string{
+				"cpu":    "128",
+				"memory": "128",
+			},
+		}
+
+		It("throws an error if the production field is missing", func() {
+			err := missingConfig.ValidateProductionFields("missing-service", "public")
+			Expect(err).To(HaveOccurred())
+			expectedErrorString := "service.yml for 'missing-service' missing required field 'production'"
```

Comment:
Thoughts on updating this error message to be
```
missing-service/service.yml missing required field 'production'
```
Would need to pass in the service location instead of the serviceName but I think that would be easier for the user to figure out where they need to fix